content/help/set-up-your-keycard-with-keycard-shell.mdx

```diff
@@ -0,0 +1,113 @@
+- [Import your key pair with a QR code](#method-3-import-your-key-pair-with-a-qr-code)
+
+A key pair consists of a public key (which generates your shareable wallet address) and a private key (which you use to sign transactions and prove ownership). One Keycard can store one master key pair, from which all derived key pairs are created. These derived key pairs corespond to different wallet addresses of your Keycard.
+
+Keycard and Keycard Shell supports BIP39 (12- or 24-word recovery phrase) and SLIP39 backup standards. SLIP39 is an advanced backup method that splits your wallet backup into multiple parts.
+
+<Admonition type="note">
+    SLIP39 is not yet widely adopted in other hardware wallets. For more information, check out [Understand the SLIP39 wallet backup standard](./understand-the-slip-39-wallet-backup-standard).
+</Admonition>
+
+When you set up a key pair, you can set a passphrase in addition to your recovery phrase. A passphrase is an optional extra word or phrase you add to your recovery phrase for extra security. If you set one, you must remember it to recover your wallet.
+
+### Method 1: Generate a new key pair
+
+1. On your Keycard Shell, select **Generate new key pair**.
+1. Choose your recovery phrase type: 12 or 24 words, or a SLIP39 backup. You can also select whether to use a passphrase. Then press **OK** to confirm the text on the screen.
+1. Proceed depending on your situation.
+
+   <Table>
+         <TableHead>
+           <TableHeader>Device to sync</TableHeader>
+           <TableHeader>Steps</TableHeader>
+         </TableHead>
+
+         <TableContent>
+           <TableRow>
+             <TableCell>12- or 24-word recovery phrase</TableCell>
+
+             <TableCell>
+               1. Write down the recovery phrase.
+               1. Press **OK**.
+             </TableCell>
+           </TableRow>
+
+           <TableRow>
+             <TableCell>SLIP39 recovery phrases</TableCell>
+
+             <TableCell>
+               1. Press ◀ ▶ **Left** / **Right** to set the total number of parts and the threshold number of parts required to recover your wallet. Then press **OK**.
+               1. Write down the recovery phrase parts.
+               1. Press **OK**.
+             </TableCell>
+           </TableRow>
+         </TableContent>
+       </Table>
+
+1. Press **OK** to confirm the text on the screen. Then confirm your recovery phrase by selecting the words as required.
+1. If you choose to set a passphrase, press ▲ ▼ **Up** / **Down** and ◀ ▶ **Left** / **Right** to select letters and set a passphrase. Press and hold **OK** to confirm.
+1. Press **OK**.
+
+<Admonition type="caution">
+    Don't take a screenshot of your recovery phrase and don't save your recovery phrase on your phone, computer or any online services. Keep your recovery phrase(s) safe and private. The Keycard team will never ask you for this information, and anyone who does is trying to steal your funds.
```

Comment:
```suggestion
    Don't take a screenshot of your recovery phrase and don't save your recovery phrase on your phone, computer, or any online services. Keep your recovery phrase(s) safe and private. The Keycard team will never ask you for this information, and anyone who does is trying to steal your funds.
```

I believe you're using the Oxford comma in the Shell documentation. Ignore this suggestion if this is not the case.